From 14439b900bd506b7fb99980cac3f62b624a8c65f Mon Sep 17 00:00:00 2001
From: Eric Sandeen <sandeen@redhat.com>
Date: Mon, 4 Aug 2014 18:58:33 -0400
Subject: [fs] btrfs: replace path->slots[0] with otherwise unused variable 'slot'

Message-id: <1407179109-13714-27-git-send-email-sandeen@redhat.com>
Patchwork-id: 86977
O-Subject: [RHEL7.1 PATCH 026/422] btrfs: replace path->slots[0] with otherwise unused variable 'slot'
Bugzilla: 1071179
RH-Acked-by: Zach Brown <zab@redhat.com>

From: Valentina Giusti <valentina.giusti@microon.de>

commit e94acd86d48d61a5d919d807ed1efa0d8c1cd5ae
Author: Valentina Giusti <valentina.giusti@microon.de>
Date:   Mon Nov 4 22:34:28 2013 +0100

    btrfs: replace path->slots[0] with otherwise unused variable 'slot'

    Signed-off-by: Valentina Giusti <valentina.giusti@microon.de>
    Signed-off-by: Josef Bacik <jbacik@fusionio.com>
    Signed-off-by: Chris Mason <clm@fb.com>
Signed-off-by: Eric Sandeen <sandeen@redhat.com>
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/fs/btrfs/backref.c b/fs/btrfs/backref.c
index 3775947..826b98c 100644
--- a/fs/btrfs/backref.c
+++ b/fs/btrfs/backref.c
@@ -1683,8 +1683,8 @@ static int iterate_inode_extrefs(u64 inum, struct btrfs_root *fs_root,
   btrfs_release_path(path);
 
   leaf = path->nodes[0];
-  item_size = btrfs_item_size_nr(leaf, path->slots[0]);
-  ptr = btrfs_item_ptr_offset(leaf, path->slots[0]);
+  item_size = btrfs_item_size_nr(leaf, slot);
+  ptr = btrfs_item_ptr_offset(leaf, slot);
   cur_offset = 0;
 
   while (cur_offset < item_size) {
-- 
1.7.1